tests/app_settings.rs

```diff
@@ -146,7 +155,9 @@ FLAGS:
 SUBCOMMANDS:
     help
             Print this message or the help of the given subcommand(s)
-    test";
+    test
+            
```

Comment:
This looks to be incorrect, will open a ticket

Reply:
also expected, given the trailing spaces on short format help messages

Reply:
What happens when there's a subcommand starting with `z`? Would appreciate you looking into it and then creating an issue with expected behaviour. In fact, there might one already for short format help messages.